commands/pack.js

```diff
@@ -62,6 +62,7 @@ function displayHelp() {
 			--externals-polyfill  	Flag whether to use external polyfill (or include in framework build)
 			--ilib-additional-path	Specify iLib additional resources path
 			--no-animation          Build without effects such as animation and shadow
+			--add-contenthash       Add the hash of the chunk, including only elements of this content type
```

Comment:
For what reason, does this need to be a private option?
If not, please make it public.

Reply:
Let's use `--content-hash` for the option name.

Reply:
Let's make the option public and add docs to explain.

Reply:
`Add the hash of the chunk, including only elements of this content type` sounds not straightforward.